Sources/Sentry/SentryScope.m

```diff
@@ -330,6 +367,13 @@ - (void)addAttachment:(SentryAttachment *)attachment
     [serializedData setValue:self.environmentString forKey:@"environment"];
     [serializedData setValue:[self fingerprints] forKey:@"fingerprint"];
 
+    [_spanLock lock];
+    id<SentrySpan> span = self.span;
+    [_spanLock unlock];
+
+    if ([span isKindOfClass:[SentryTracer class]])
+        [serializedData setValue:[(SentryTracer *)span name] forKey:@"transaction"];
```

Comment:
why do we need to serialize the span?

Reply:
The only place in the SDK that uses the serialization of the scope is to populate SentryCrashIntegration. 
I guess the transaction name is a good think to have on crash information.

Reply:
the association of the event and transaction is already thru `event.contexts.trace` so the transaction name is already available on Sentry dashboard